src/entities/delete.js

```diff
@@ -176,6 +177,18 @@ async function deleteEntities(entities, options = {}) {
             }
         });
     }
+
+    if (options.waitSubmitted) {
+
+        // wait for scene ops to finish
+        await new Promise((resolve) => {
+            if (api.realtime.scenes.current) {
+                api.realtime.scenes.current.whenNothingPending(resolve);
```

Comment:
Is this current API? `whenNothingPending` is a bit of a mouthful 😅 What does this do?

Reply:
https://share.github.io/sharedb/api/doc#whennothingpending

Reply:
Oh, that's SharedDB? That is a mouthful and a half

Reply:
https://share.github.io/sharedb/api/doc#whennothingpending

Reply:
added a better (hopefully) comment